examples/pyfar_demo.ipynb

```diff
@@ -1,6 +1,8 @@
 {
```

Comment:
how can I remove the changes made to this files?

Reply:
You can copy the version from develop and commit this to your branch :)